Gateway/charts/gateway/values.yaml

```diff
@@ -21,7 +21,7 @@ ingress:
     kubernetes.io/ingress.class: addon-http-application-routing
   path: /
   hosts:
-  - dev.gateway.<UPDATE_ME>
+  - dev.gateway.<REPLACE_ME_BY_HOST_SUFFIX>
```

Comment:
nit: I think "<REPLACE_ME_WITH_HOST_SUFFIX>"

Reply:
Sure! On it